templates/fargate.cfn.yml

```diff
@@ -715,6 +736,31 @@ Resources:
     - DefaultTargetGroup
     - ApplicationLoadBalancer
 
+  LBLatency:
+    Type: 'AWS::CloudWatch::Alarm'
+    Properties: 
+      AlarmName: LoadBalancer Latency Alarm
+      AlarmDescription: Average load balancer latency
```

Comment:
The alarm description should clarify the conditions.

E.g.,

AlarmDescription: !Sub DB insert latency is over ${DatabaseAlarmWriteLatencyMaxSeconds} for ${DatabaseAlarmEvaluationPeriods} period(s) of ${DatabaseAlarmEvaluationPeriodSeconds} seconds